packages/package.go

```diff
@@ -387,6 +388,20 @@ func (p *Package) HasKibanaVersion(version *semver.Version) bool {
 	return p.Conditions.Kibana.constraint.Check(version)
 }
 
+func (p *Package) HasCapabilities(capabilities []string) bool {
```

Comment:
Maybe we should use a different name. This could refer to the fact that the package requires the passed capabilties to properly work.
```suggestion
func (p *Package) WorksWithCapabilities(capabilities []string) bool {
```